pkg/volume/hostpath/host_path_test.go

```diff
@@ -172,7 +172,14 @@ func TestProvisioner(t *testing.T) {
 	if err != nil {
 		t.Errorf("Failed to make a new Provisioner: %v", err)
 	}
-	pv, err := creator.Provision(nil, nil)
+
+	hostPathCreator, ok := creator.(*hostPathProvisioner)
+	if !ok {
+		t.Error("Not a hostPathProvisioner")
+	}
+	hostPathCreator.basePath = fmt.Sprintf("%s.%s", "hostPath_pv", uuid.NewUUID())
```

Comment:
Should this match with `tempPath`?

Reply:
> Should this match with `tempPath`?

I think should not. This is for `hostpath_pv`, `tempPath` is for `hostpath`